build.sbt

```diff
@@ -99,13 +99,22 @@ lazy val parsing = project("akka-parsing")
 
 lazy val httpCore = project("akka-http-core")
   .settings(AutomaticModuleName.settings("akka.http.core"))
-  .dependsOn(parsing)
+  .dependsOn(httpModel)
   .addAkkaModuleDependency("akka-stream", "provided")
   .addAkkaModuleDependency("akka-stream-testkit", "test")
   .settings(Dependencies.httpCore)
   .settings(VersionGenerator.versionSettings)
   .enablePlugins(BootstrapGenjavadoc)
 
+lazy val httpModel = project("akka-http-model")
```

Comment:
- Maybe this could come before `httpCore`:
  - a tiny bit more sequential in terms of dependency, `httpModel` depends on `parsing` and `httpCore` depends on `httpModel`.

- Should the `root` project aggregate `httpModel` as well?